steps/common/steps.go

```diff
@@ -56,6 +60,24 @@ func (cs Steps) InitializeSteps(ctx context.Context, scenCtx *godog.ScenarioCont
 		}
 		return fmt.Errorf("mismatch of values: expected '%s', actual '%s'", e, v)
 	})
+	scenCtx.Step(`^I store my ip( (?:[0-9]{1,3}\.){3}[0-9]{1,3})? in context "([^"]*)"$`, func(desiredIP, key string) error {
```

Comment:
I guess it makes no sense to store an specific IP in context with this step. That's an specific operation you can do saving whatever you want in context with:

`^I store "([^"]*)" in context "([^"]*)"$`

So that I will leave the initial proposal, with a minimal change in nomenclature:

`^I store own ip in context "([^"]*)"$`